components/layout/display_list/builder.rs

```diff
@@ -3161,12 +3161,12 @@ impl ComputedValuesCursorUtility for ComputedValues {
     #[inline]
     fn get_cursor(&self, default_cursor: CursorKind) -> Option<CursorKind> {
         match (
-            self.get_pointing().pointer_events,
-            self.get_pointing().cursor,
+            &self.get_pointing().pointer_events,
```

Comment:
I'm getting a `cannot move out of borrowed content` error here. We can solve it in two ways: by borrowing like here and then pattern matching the references or by copying like so: 
```Rust
self.get_pointing().pointer_events.clone()
```
and leaving the rest the way it is. Should we borrow or copy?